buildSrc/src/main/groovy/io.deephaven.java-toolchain-conventions.gradle

```diff
@@ -125,6 +125,14 @@ def parseJvmArgumentsFromEnv = { String envName, List<String> orElse ->
   return s.split('\\s+').findAll { it -> !it.trim().isEmpty() } as List<String>
 }
 
+def parseJvmArgumentsFromProperty = { String propertyName, List<String> orElse ->
+  if (!project.hasProperty(propertyName)) {
+    return orElse
+  }
+  // Note: this logic could be improved to handle quotes and other cases, but hopefully this suffices for now.
+  return project.property(propertyName).toString().split('\\s+').findAll { it -> !it.trim().isEmpty() } as List<String>
```

Comment:
i think this might be technically wrong, since you could pass `-Dfoo="hello, world"`, and wouldn't expect that to be split into two params.

Reply:
I agree, and that's what the comment is about. The same limitation exists in `parseJvmArgumentsFromEnv`:

```shell
START_OPTS='-Dfoo="hello, world"' ./gradlew server-jetty-app:run
./gradlew server-jetty-app:run -Pdeephaven.startOpts='-Dfoo="hello, world"'
```

both fail in the same way. For setting Deephaven configuration properties, I would recommend the Deephaven configuration file. I could see a legitimate place where this would fail if you are setting some JVM argument that takes in a filename with spaces (for example, trying to log out GC to a path on the filesystem that has a space in it). That said, it only effects developers, and these sites could be improved in the future if necessary.